SPEC.md

```diff
@@ -94,6 +94,7 @@
   * [Array\[Array\[X\]\] transpose(Array\[Array\[X\]\])](#arrayarrayx-transposearrayarrayx)
   * [Pair(X,Y) zip(X,Y)](#pairxy-zipxy)
   * [Pair(X,Y) cross(X,Y)](#pairxy-crossxy)
+  * [String prefix(String, Array\[String\])](#string-prefix-string-arraystring)
```

Comment:
Whoops, that's because I have a superflous dash.  Thank you for reviewing so thoroughly!  😄